packages/atclient/include/atclient/atclient.h

```diff
@@ -153,7 +153,7 @@ int atclient_get_publickey(atclient *atclient, atclient_connection *root_conn, c
  * should be created (if it hasn’t already been) or not.
  * @return int 0 on success
  */
-int atclient_get_sharedkey(atclient *atclient, const atclient_atkey *atkey, char *value, const size_t valuelen,
```

Comment:
can't do const with these, since the atkey needs to be written to by certain underlying functions